extensions/geode-modules/src/main/java/org/apache/geode/modules/session/catalina/DeltaSession.java

```diff
@@ -586,6 +610,20 @@ public int getSizeInBytes() {
     return serializedAttributes;
   }
 
+  protected ConcurrentMap getAttributes() {
+    try {
+      Field field = getAttributesFieldObject();
+      field.setAccessible(true);
+      Map oldMap = (Map) field.get(this);
+      ConcurrentMap newMap = new ConcurrentHashMap();
```

Comment:
I think you are correct, I believe we wrapped it incase somehow it was not a concurrent hash map but I guess that was being overly cautious.  I'll change them all to map